doas: add page (#3939)
tldr-pages/tldr: pages/common/doas.md

# doas

> > Executes a command as another user.

Reviewer: hi @user56441 can you provider some document link here?

- Run a command as root:

`doas {{command}}`

- Run a command as another user:

`doas -u {{user}} {{command}}`

- Launch the default shell as root:

`doas -s`

> - Parse and check a config file to check if the command is permited or denied:

Reviewer 2: ```suggestion
- Parse a config file and check if the command is allowed:
```

`doas -C {{config_file}} {{command}}`

> - Make doas request for a password even after supplied earlier:

Reviewer 2: ```suggestion
- Make doas request for a password even after it was supplied earlier:
```
